docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -1016,7 +991,9 @@ The following table lists all the server state attributes the client can receive
 | `MC_PLAYBACK_STATE_MOVING_TO_PREVIOUS` | Playback state of moving to previous media (Tizen 4.0 and Higher for Mobile and Tizen 5.0 and Higher for Wearable) |
 | `MC_PLAYBACK_STATE_FAST_FORWARDING` | Playback state of fast forwarding (Tizen 4.0 and Higher for Mobile and Tizen 5.0 and Higher for Wearable) |
 | `MC_PLAYBACK_STATE_REWINDING`       | Playback state of rewinding (Tizen 4.0 and Higher for Mobile and Tizen 5.0 and Higher for Wearable) |
-
+| `MC_PLAYBACK_STATE_CONNECTING`  | Playback state of connecting (Tizen 6.0 and Higher)         |
+| `MC_PLAYBACK_STATE_BUFFERING`   | Playback state of buffering (Tizen 6.0 and Higher)          |
+| `MC_PLAYBACK_STATE_ERROR`       | Playback state of error  (Tizen 6.0 and Higher)             |
```

Comment:
@jiyong2 

Please remove extra space between error    (Tizen 6.0...
| `MC_PLAYBACK_STATE_ERROR`       | Playback state of error (Tizen 6.0 and Higher)           |

Reply:
oh, it was removed.